lib/jxl/modular/transform/squeeze.cc

```diff
@@ -119,8 +119,7 @@ Status InvHSqueeze(Image &input, uint32_t c, uint32_t rc, ThreadPool *pool) {
       pixel_type left = (x ? p_out[(x << 1) - 1] : avg);
       pixel_type tendency = SmoothTendency(left, avg, next_avg);
       pixel_type diff = diff_minus_tendency + tendency;
-      pixel_type A =
-          ((avg * 2) + diff + (diff > 0 ? -(diff & 1) : (diff & 1))) >> 1;
+      pixel_type A = avg + (diff / 2);
```

Comment:
There's a similar line in InvVSqueeze, can do the same simplification there.

Reply:
Sure. One moment.